Libraries/Alert/Alert.js

```diff
@@ -31,6 +32,11 @@ export type Buttons = Array<{
 type Options = {
   cancelable?: ?boolean,
   onDismiss?: ?() => void,
+  surface?: mixed,
```

Comment:
I struggle to come up with the right name for this argument (also in other modified APIs).

 Taking @shergin's suggestions, it's not `rootTag` anymore, since: a) we don't really need the root, any component/view will do, and b) it shouldn't be a numeric tag for fabric forward compatibility.

Instead, I'm passing this to `ReactNative.findNodeHandle()` -- so a `ref.current` to any component will do, but you _can_ also pass a numeric tag if you prefer.